test_main.py

```diff
@@ -19,3 +19,16 @@ def test_positionof():
   assert positionof("bitten") == 16
   assert positionof("id") == 0
 # There isn't actually any other tests yet
+
+# Check if the kill_queue is fully functional
+import management.db as db
+import reset
+
+def kill_queue_test():
+  db.add_kill(12738912739821,"Barber")
+  db.add_kill(12347892374923,"White Werewolf",7289347983274)
+  print(db.get_kill())
+  print(db.get_kill())
+  print(db.get_kill())
+  reset.reset(True)
+  return True
```

Comment:
This is completely useless, the tests, by standards, shouldn't be printing anything. Use an `assert` statement to make sure `get_kill` is returning the right thing. You should also test that `reset.reset` works. I suggest making it return True if the reset is successful, and the error if else.